### PR TITLE
Fix #650: added InputPhone

### PR DESCRIPTION
**Pull request by jepsar**

### CODE REVIEW

**Review by melloware — CHANGES_REQUESTED** [review]
Don't forget you need to add it to the taglib.xml file too and all its properties.

File: src/main/java/org/primefaces/extensions/component/inputphone/Country.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * limitations under the License.
+ */
+package org.primefaces.extensions.component.inputphone;
+
+/**
+ *
+ * @author Jasper de Vries &lt;jepsar@gmail.com&gt;
+ */
+public class Country {
+
```
Comment (melloware) [inline src/main/java/org/primefaces/extensions/component/inputphone/Country.java]: Best to make this implement Serializable and add an equals(), toString(), and hashCode(). Whenevre you have these POJO's you never know how someone might use them or put them in SessionScoped beans etc.

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by jepsar** [comment]
> Don't forget you need to add it to the taglib.xml file too and all its properties.

Sorry, I'm not sure what you mean. I modified primefaces-extensions.taglib.xml. Did I miss something?

**Review by tandraschko — COMMENTED** [review]
In pf core we would put this class into the model package

**Comment by jepsar** [comment]
> In pf core we would put this class into the model package

I wasn't sure about this.. I'll move it.

**Comment by melloware** [comment]
@jepsar Nice work.  For some reason the taglib was now showing up in my Diff but it is now!  Thanks.  I will merge tonight and send you a note when i have it all working with minifying everything correctly.

**Review by melloware — APPROVED** [review]

**Pull request merged**